test/TestProjects/SubClients-LowLevel/Generated/Parameter.cs

```diff
@@ -49,6 +49,18 @@ internal Parameter(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, A
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="subParameter"/> is null. </exception>
         /// <exception cref="ArgumentException"> <paramref name="subParameter"/> is an empty string, and was expected to be non-empty. </exception>
+        /// <example>
```

Comment:
Exceptions above should include RequestFailedException.

Reply:
I don't know why github doesn't show `outdated` for this file. Actually the latest version contains `RequestFailedException`.